pkg/agent/loadbalancer/loadbalancer.go

```diff
@@ -154,3 +155,9 @@ func onDialError(src net.Conn, dstDialErr error) {
 	logrus.Debugf("Incoming conn %v, error dialing load balancer servers: %v", src.RemoteAddr().String(), dstDialErr)
 	src.Close()
 }
+
+// ResetLoadBalancer will delete the local state file for the load balacner on disk
+func ResetLoadBalancer(dataDir, serviceName string) {
+	stateFile := filepath.Join(dataDir, "etc", serviceName+".json")
+	os.Remove(stateFile)
```

Comment:
Can we catch the error and at least log it?

Reply:
sure